providers.tf

```diff
@@ -1,4 +1,3 @@
 provider "aws" {
-  alias  = "owner"
-  region = module.this.aws_region
```

Comment:
didn't you say something like the region was required when setting an alias or so?

Reply:
Yes but now you have a different issue as it is duplicated if you define that part by yourself

Reply:
Ahh i see.